src/main/java/redis/clients/jedis/Jedis.java

```diff
@@ -8514,6 +8521,13 @@ public String clientInfo() {
     return connection.getBulkReply();
   }
 
+  @Override
+  public String clientSetInfo(ClientAttributeOption attr, String value) {
+    checkIsInMultiOrPipeline();
+    connection.sendCommand(CLIENT, SETINFO.getRaw(), attr.getRaw(), encode(value));
+    return connection.getStatusCodeReply();
+  }
+
```

Comment:
[Discussion]

@yangbodong22011 WDYT about removing this support from Jedis class? Meaning to keep the usages of SETINFO only internally.

https://github.com/redis/redis/pull/11758 mentions it is for _Client libraries_, not necessarily for users.

Reply:
I hope to keep the status quo. Although users do not need to actively set it in most cases, if users want to force it, we can still support it. And `redis-cli` or `sendCommand` can also accomplish the same function.